docs/sphinx/manuals/roc_recv.rst

```diff
@@ -65,6 +66,13 @@ Driver
 
 If the driver is omitted, some default driver is selected. If the user did specify the output and it is a file with a known extension, the appropriate file driver is selected. Otherwise, the first device driver available on the system is selected.
 
+Multicast
+---------
```

Comment:
Let's move this section after "Port" section.

Reply:
Done

Reply:
Now it's in after the EXAMPLES sections. And I meant after the Port section: https://github.com/roc-project/roc/blob/2aa69410aa32a782eab61c71f39a9128235cfb4f/docs/sphinx/manuals/roc_recv.rst#port

Reply:
a29a82809240bdc12d8996d7112b882ff9fa0e9a